### PR TITLE
Add Omni bridge details.

### PR DESCRIPTION
**Pull request by adamiak**
Resolves L2B-203

### CODE REVIEW

**Review by luke7211 — CHANGES_REQUESTED** [review]
Expand section on "investing" bridge funds

**Comment by sz-piotr** [comment]
I have update the risk view descriptions to be in line with https://github.com/l2beat/l2beat/pull/794. Please check if the changes are correct before proceeding

File: packages/config/src/tokens.ts [hunk 1]
```diff
@@ -500,6 +500,16 @@ export const tokenList: TokenInfo[] = [
     sinceTimestamp: new UnixTime(1561649739),
     category: 'other',
   },
+  {
```
Comment (sz-piotr) [inline packages/config/src/tokens.ts:503]: Remove all new tokens because they do not comply with our policy

**Review by sz-piotr — CHANGES_REQUESTED** [review]

File: packages/config/src/bridges/omni.ts [hunk 2]
```diff
@@ -1,40 +1,209 @@
...
+      description: '4/6 Validator MultiSig',
+      sentiment: 'bad',
+    },
+    sourceUpgradeability: {
+      value: 'Yes',
+      description: 'Contracts can be upgraded by 7/16 MultiSig',
+      sentiment: 'bad',
+    },
+    destinationToken: {
+      ...RISK_VIEW.WRAPPED,
```
Comment (sz-piotr) [inline packages/config/src/bridges/omni.ts:61]: canonical

**Review by sz-piotr — CHANGES_REQUESTED** [review]

**Review by sz-piotr — APPROVED** [review]

**Pull request merged**